docker-compose.yml

```diff
@@ -81,8 +82,9 @@ services:
       SCHEDULER_OUTPUT_MODE: UpdateInputPlanWithNewActivities
       SCHEDULER_PORT: 27193
       SCHEDULER_RULES_JAR: /usr/src/app/merlin_file_store/scheduler_rules.jar
+      JAVA_OPTS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
     image: aerie_scheduler
-    ports: ["27193:27193"]
+    ports: ["27193:27193", "5006:5005"]
```

Comment:
Is this supposed to be `5006:5005`? I would've expected `5006:5006` I think.

Reply:
Yeah - it just needs to match the port in the JAVA_OPTS line above it. I can set the JAVA_OPTS to use 5006 and update this line accordingly - that would probably be less confusing.

Reply:
Ohh! Got it, thanks. No worries this is all good :)